### PR TITLE
update memory stats examples links

### PR DESCRIPTION
**Pull request by maciejbocianski**
examples move PR: https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/83 https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/88

### CODE REVIEW

File: docs/api/memory/MemTrace.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ For a step-by-step guide about how to use optimize memory using runtime memory t
 
 ## Memory tracing example
 
-[![View code](https://www.mbed.com/embed/?url=https://os.mbed.com/teams/mbed_example/code/memory_tracing_example/)](https://os.mbed.com/teams/mbed_example/code/memory_tracing_example/file/168ab14e6694/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_Platform/Memory_tracing_ex_1/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_Platform/Memory_tracing_ex_1/main.cpp)
```
Comment (AnotherButler) [inline docs/api/memory/MemTrace.md:27]: I don't see this change in the mbed-os-examples-docs_only PR.

**Comment by jamesbeyond** [comment]
Code merged

File: docs/api/memory/MemTrace.md (hunk 1 above)
Comment (maciejbocianski) [reply docs/api/memory/MemTrace.md:27]: example is here https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/88

**Review by AnotherButler — APPROVED** [review]
Thanks for the PR 👍

**Comment by jamesbeyond** [comment]
code change merged

**Pull request merged**